core/themes/basis/template.php

```diff
@@ -38,6 +38,18 @@ function basis_preprocess_page(&$variables) {
     $variables['classes'][] = 'view-name-' . $view->name;
   }
 
+  // Add body class for each upadte.
+  $install_version = config_get('system.core', 'initial_version');
+  // Every time we add supplemental CSS we add the version number here.
+  $updated_versions = array('1.30.0');
```

Comment:
I find both variable names a bit misleading. Maybe a more detailed comment might help. Or different variable names... "initial_version" and "supplemental_css_version"?